swim/labels_test.go

```diff
@@ -1,12 +1,98 @@
 package swim
 
 import (
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+var (
```

Comment:
Thanks, this uncovered that I missed the configuration propagation from ringpop to the memberlist. The tests can be found here: https://github.com/uber/ringpop-go/pull/171/files#diff-2800ccf278cea772a67ec71a391a7761R866